feature/bgp/bgp_isis_redistribution/otg_tests/bgp_isis_redistribution_test/bgp_isis_redistribution_test.go

```diff
@@ -357,6 +362,14 @@ func nonMatchingCommunityRoutePolicy(t *testing.T, dut *ondatra.DUTDevice) {
 		} else {
 			stmt.GetOrCreateConditions().GetOrCreateBgpConditions().GetOrCreateMatchCommunitySet().SetCommunitySet(v4CommunitySet)
 		}
+		// Configure ALLOWAll policy
+		pdef = rp.GetOrCreatePolicyDefinition(allowAllPolicy)
+		stmt, err = pdef.AppendNewStatement("id-1")
+		if err != nil {
+			t.Fatalf("AppendNewStatement(%s) failed: %v", "id-1", err)
```

Comment:
What is the need to create this additional statement?
The routing policy and policy statements are created above in ln342; could we leverage this code?

Reply:
allowAllPolicy is referred in bgp import/export policy in function setupEBGPAndAdvertise.
This policy config was getting replaced in line 360, hence added back allowAllPolicy

Reply:
Ok; could we do a gnmi.update in setupEBGPAndAdvertise function? in that case we do not have add this part of the code.

Reply:
setupEBGPAndAdvertise function is called in initial config before the testcases, modifying to gnmi.update won't be helpful here. Function nonMatchingCommunityRoutePolicy  as part of third testcase 'NonMatchingCommunity' does gnmi.Replace of policy config  which removes allowAllPolicy and hence had to add it  back .